truckersmp_cli/steamcmd.py

```diff
@@ -130,26 +130,38 @@ def update_game():
             env=env_steam)
 
     if Args.proton:
-        # download/update Proton
-        os.makedirs(Args.protondir, exist_ok=True)
-        logging.debug("Updating Proton (AppID:%s)", Args.proton_appid)
-        logging.info(
-            """Command:
+        update_proton = True
+        if Args.skip_update_proton:
+            # skip updating Proton only when Proton is already installed
+            try:
+                with open(os.path.join(Args.protondir, "proton")):
+                    pass
+            except OSError:
+                logging.info("Proton is not installed yet, installing Proton")
```

Comment:
This does completely the opposite as the argument suspects.
Checking and failing before starting would be better imho.
We already have a similar check in place which could be expanded.